src/data/cached-history.ts

```diff
@@ -121,6 +121,10 @@ export const getRecentWithCache = (
 
   const curCacheProm = cache.prom;
 
+  // For accurate history display of an input_datetime we require the attributes
+  // => we need a complete response => minimalResponse = false
+  const minimalResponse = entityId.split(".")[0] !== "input_datetime";
```

Comment:
How would we do this if we are getting the history of multiple entities?

It is actually handled in the backend:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/history/__init__.py#L71-L76

Reply:
Good catch. Core PR is on its way. Then from the looks of it, this frontend PR can be completely rejected (and I could have saved myself a lot of time, had I seen the backend exclusion list).